CHANGES

```diff
@@ -60,21 +73,31 @@ Lib/
 Lib/slapdtest.py
 * Automatically try some common locations for SCHEMADIR
 * Ensure server is stopped when the process exits
+* Check for LDAP schema and slapd binaries
```

Comment:
slapdtest also includes certificates (CA, server, client) and configuration for TLS client auth.

Apropos certificates, the test certs are **not** installed with ``slapdtest``. Is slapdtest shipped in standard distributions?

Reply:
It is in the sdist, yes.

Reply:
But actually, it's python-ldap's test suite that includes the certificates, not not `slapdtest.py`.
This should be fine.

Reply:
Fixing in https://github.com/python-ldap/python-ldap/pull/73

Reply:
thanks, I opened #76 to track the issue.